test/SampleWebAPIIntegration/SampleWebAPIIntegration.xproj

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This project contains what we call functional tests. Do we want the name of the project to reflect that? WebAPI.FunctionalTests perhaps?